karma.conf.js

```diff
@@ -15,25 +15,36 @@ const browsers = ci
     ? undefined
     : watch
       ? ['Chrome']
-      : ['Chrome', 'Firefox']
+      : ['ChromeHeadless', 'FirefoxHeadless']
 
 module.exports = function (config) {
   config.set({
     basePath: '.',
-    frameworks: ['mocha'],
+    frameworks: ['mocha', 'karma-typescript'],
```

Comment:
I don't like karma-typescript or most of these kinds of creations. I see them as unnecessary couplings. I like chaining my tools myself. It seems simpler to me and with less of the pipeline under the hood of some dependency.